### ISSUE TITLE
Preprocess the Data

### ISSUE THREAD

**Issue by Abel2** [ZED-AI-Lab/zed-ai-lab-arrant-engines#3]
Pre-process your data by running commands  such as 

## Normalize pixel values to range 0–1
x_train = x_train / 255.0
x_test = x_test / 255.0

this allwo you to normalize  and encode your data(one-hot encode)
# One-hot encode the labels
from tensorflow.keras.utils import to_categorical
y_train = to_categorical(y_train)
y_test = to_categorical(y_test)
